apps/web/src/components/molecules/cardActions/EventCardActions.tsx

```diff
@@ -18,8 +19,32 @@ export const EventCardActionsFuture: React.FC<{ event: EventResponse }> = ({
 
   const timeLeft = dayjs().to(event.from, true);
 
+  function parseDataTime(date: string) {
+    // 20231130T123900 - format
+    return dayjs(date).format("YYYYMMDDTHHmmss");
+  }
+
   return (
     <div className={styles.actions}>
+      <div
+        style={{
+          maxHeight: "100px",
+        }}
+      >
+        <BasicCallendarButton
+          name={event.title}
+          startTime={parseDataTime(event.from)}
+          endTime={parseDataTime(event.to)}
```

Comment:
tu proponowałbym przekazywać parametr normalnie jako event.from i event.to a parsować dopiero w komponencie, w ten sposób to sam komponent będzie odpowiedzialny za odpowiednie sformatowanie daty